spec/swagger_v2/params_array_spec.rb

```diff
@@ -118,8 +118,7 @@
         end
 
         specify do
-          expect(subject['definitions']['postArrayOfType']['type']).to eql 'array'
-          expect(subject['definitions']['postArrayOfType']['items']).to eql(
```

Comment:
This confused me:
```
          # as body parameters it would be interpreted a bit different,
          # cause it could not be distinguished anymore, so this would be translated to one array,
          # see also next example for the difference
          params do
            requires :array_of_string, type: Array[String], documentation: { param_type: 'body', desc: 'nested array of strings', example: %w[a b] }
            requires :array_of_integer, type: Array[Integer], documentation: { param_type: 'body', desc: 'nested array of integers' }
          end
```
I still do not understand why this had an array type in a root and it was "okay". 🤔

Reply:
if I remember me correctly, then grape allows to define an Array of type in first place, but as you described, this is not the way OAPI handle it

means grape allows much more then OAPI2.x and we have to deal with it, it will change with support of OAPI3.x, but there is no progress atm

Reply:
sorry, we wrote at the same time …

so think adding also a comment into the UPGRADE.md to let others know, that there is a possible breaking change

Reply:
OAPI3.x 😍